package/MDAnalysis/guesser/DefaultGuesser.py

```diff
@@ -0,0 +1,155 @@
+from .base import GuesserBase
+import numpy as np
+import warnings
+
+import re
+
+from ..lib import distances
+from . import tables
+
+
+class DefaultGuesser(GuesserBase):
+    context = 'default'
+ 
+    def __init__(self, atoms):
+        super().__init__(atoms)
+        self._guess = {'mass': self.guess_masses,
+                       'type': self.guess_types}
+        self._rank = {'mass': 2,
+                      'type': 1
+                      }
+
+    def guess_masses(self):
+        """Guess the mass of many atoms based upon their type
+
+        Returns
+        -------
+        atom_masses : np.ndarray dtype float64
```

Comment:
Does this format nicely when the type is given this way?

pleas also add a sentence description.